build.gradle

```diff
@@ -4,43 +4,43 @@ buildscript {
     ext {
         android_version_code = 1
         android_version_name = "0.1.0"
-        android_build_tools_version = "27.0.3"
```

Comment:
am I missing sth or this has been removed entirely instead of replacement somewhere? 🤔

Reply:
@sleepylee I think build tools version is not necessary anymore from gradle 3.1.0. More info https://developer.android.com/studio/releases/gradle-plugin#3-1-0
```
Keep in mind, you no longer need to specify a version for the build tools using the android.buildToolsVersion property—the plugin uses the minimum required version by default.
```